### PR TITLE
Create FAT tests for @OpenIdProviderMetadata

### PR DESCRIPTION
**Pull request by c00crane**
Create FAT tests for @OpenIdProviderMetadata

For #21158

### CODE REVIEW

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,io.openliberty.security.jakartasec.3.0.internal_fat.config,io.openliberty.security.jakartasec.3.0.internal_fat.logout,io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,io.openliberty.security.jakartasec.3.0.internal_fat.config,io.openliberty.security.jakartasec.3.0.internal_fat.logout,io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Comment by c00crane** [comment]
Requesting one more build since the previous one ran into a no more space on device issue:
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,io.openliberty.security.jakartasec.3.0.internal_fat.config,io.openliberty.security.jakartasec.3.0.internal_fat.logout,io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Comment by c00crane** [comment]
Requesting yet one more build since the previous one also ran into a no more space on device issue:
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,io.openliberty.security.jakartasec.3.0.internal_fat.config,io.openliberty.security.jakartasec.3.0.internal_fat.logout,io.openliberty.security.jakartasec.3.0.internal_fat.refresh

**Comment by c00crane** [comment]
#libby
#build
#spawn.fullfat.buckets=io.openliberty.security.jakartasec.3.0.internal_fat,io.openliberty.security.jakartasec.3.0.internal_fat.config,io.openliberty.security.jakartasec.3.0.internal_fat.logout,io.openliberty.security.jakartasec.3.0.internal_fat.refresh

File: dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/Endpoints.war/src/endpoints/servlets/GenericTestServlet.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        doWorker(request, response, Constants.GETMETHOD);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        doWorker(request, response, Constants.POSTMETHOD);
+    }
+
+    protected void doWorker(HttpServletRequest request, HttpServletResponse response, String method) throws ServletException, IOException {
```
Comment (ayoho) [inline dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/Endpoints.war/src/endpoints/servlets/GenericTestServlet.java:45]: `HttpServletRequest` has a `getMethod()` method that will return the HTTP method being used, so you don't strictly need to have a `method` argument here.

**Review by ayoho — DISMISSED** [review]

File: dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/Endpoints.war/src/endpoints/servlets/GenericTestServlet.java (hunk 1 above)
Comment (c00crane) [reply dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/Endpoints.war/src/endpoints/servlets/GenericTestServlet.java:45]: ok - will include that change in what I'm working on now

**Comment by c00crane** [comment]
#libby
#build

**Review by ayoho — APPROVED** [review]

**Pull request merged**